libsel4/sel4_plat_include/star64/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2023, Ivan Velickovic
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Maybe 'BSD-2-Clause' would be better here, as this links with user code that may have a problem then due to GPL infection. Or use Dual-Licensing.

Reply:
Yes, I should change that to be consistent with all the other platform constant headers. Thanks.